tests/utils/inputparams/testparams.go

```diff
@@ -96,52 +101,65 @@ func GetEndPoint2() string {
 
 // GetSwarmManager1 returns swarm manager node IP from the configured swarm cluster
 func GetSwarmManager1() string {
-	return os.Getenv("MANAGER1")
+	manager1 := os.Getenv("MANAGER1")
+	if manager1 == "" {
+		log.Printf("Manager node not found. Need this to run swarm test.")
```

Comment:
That's not the case. After addressing #1515, swarm test and other complicated tests will be moved out of the basic precheckin test suite. So it doesn't matter if we use log.Fatal() or not here. When swarm test and other complicated tests are REQUIRED to run - for example, in CI - we still want to fail the whole test if these environment variables are not available.